### DISCUSSION TITLE
"Forbidden (403): Insufficient privileges to complete the operation" error with Add-PnPMicrosoft365GroupMember and Add-PnPMicrosoft365Groupowner

### DISCUSSION THREAD

**Discussion by rashian61** [pnp/powershell#4936] [General]
### Discussed in https://github.com/pnp/powershell/discussions/4276

<div type='discussions-op-text'>

<sup>Originally posted by **rashian61** September 12, 2024</sup>
Hi - I am encountering a 'Forbidden (403): **Insufficient privileges to complete the operation**' error with Add-PnPMicrosoft365GroupMember and Add-PnPMicrosoft365GroupOwner after creating the PnP PowerShell EntraID app (https://pnp.github.io/blog/post/changes-pnp-management-shell-registration). I can create an Office365 group using New-PnPMicrosoft365Group, create new Team sites using New-PnPSite, and also remove a Microsoft365 group using Remove-PnPMicrosoft365Group. Are there any additional permissions needed? or is this a bug?  PowerShell version 7.4.5 and PnP.Power module version 2.12.0. Thanks 

![Pnp_1](https://github.com/user-attachments/assets/ba22b707-e94b-4f36-bb9e-d0e4e0d0af92)
![Pnp_2](https://github.com/user-attachments/assets/2468c8ed-2dba-4124-a65c-5a5b3e4bfcf8)

</div>

**Comment by readmo123** [comment]
The type needs to be delegated for the two MS Graph permissions.
